pgml-extension/src/bindings/transformers/transformers.py

```diff
@@ -197,7 +197,7 @@ def create_pipeline(task):
         lower = model_name.lower()
     else:
         lower = None
-    if lower and "-ggml" in lower:
+    if lower and ("-ggml" in lower or "-gguf" in lower):
```

Comment:
@kczimm these run on the CPU, so could be useful for local dev on a M1. I'll try on my end just to be sure (on my desktop right now)

Reply:
Worked on my M1 after uninstalling `deepspeed`.